configs/config.yaml

```diff
@@ -0,0 +1,161 @@
+#
+# Application configuration file
+#
+
+# application environment (dev, staging, prod)
+environment: dev
+
+coordinator:
+  # address if the server want to connect directly to debug
+  debugHost:
+  # games library
+  library:
+    # some directory which is gonna be the root folder for the library
+    # where games are stored
+    basePath: assets/games
+    # an explicit list of supported file extensions
+    # which overrides Libretro emulator ROMs configs
+    supported:
+    # a list of ignored words in the ROM filenames
+    ignored:
+      - neogeo
+      - pgm
+    # print some additional info
+    verbose: true
+    # enable library directory live reload
+    # (experimental)
+    watchMode: false
+  monitoring:
+    port: 6601
+    # enable Go profiler HTTP server
+    profilingEnabled: false
+    metricEnabled: false
+    urlPrefix: /coordinator
+  # the public domain of the coordinator
+  publicDomain: http://localhost:8000
+  # specify the worker address that the client can ping (with protocol and port)
+  pingServer:
+  # HTTP(S) server config
+  server:
+    port: 8000
+    httpsPort: 443
+    httpsKey:
+    httpsChain:
+
+worker:
+  network:
+    # a coordinator address to connect to
+    coordinatorAddress: localhost:8000
+    # ISO Alpha-2 country code to group workers by zones
+    zone:
```

Comment:
this `zone` and `httpsKey`, `httpsChain` is only for production right? I can try new deployment with these flags?

Reply:
I didn't change any related logic, so yes, it's for your DO.
It will be great if you try it but I forgot to add environment vars into that for convenient use with Docker.
The chain of config params should be: file -> (almost done, next PR -- config requested from coordinator if no file) -> environment vars -> comandline arguments.